app/feature/feature-payments/src/main/kotlin/com/hedvig/android/feature/payments2/navigation/PaymentsGraph.kt

```diff
@@ -65,8 +68,8 @@ fun NavGraphBuilder.paymentsGraph(
     composable<PaymentsDestinations2.History> {
       PaymentHistoryDestination(
         paymentOverview = paymentOverview.copy(
-          paymentConnection = null,
-        ), // Payment connection is not valid for historic payments
+          paymentConnection = null, // Payment connection is not valid for historic payments
```

Comment:
Could this be a hint that we want a different type to be sent over at `PaymentDestination2.History`, if we need to manually in the UI be careful to make something null, but still have access to it? Even though we may not really need to have access to it in here?

Reply:
Yes you are right. We should probably observe the state from the view model more instead of passing around the objects. Ill make some changes on a PR on top of everything to avoid too many conflicts.